yolact++_colab_demo.ipynb

```diff
@@ -0,0 +1,392 @@
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "bXBoJjdYuNCB",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!gdown \"https://drive.google.com/uc?id=15id0Qq5eqRbkD-N3ZjDZXdCvRyIaHpFB&export=download\" -O \"./weights/yolact_plus_base_54_800000.pth\""
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SZt7kNerxgOJ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "cd external/DCNv2"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mu-UK-Of-N8F",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Compile deformable convolutional layers (from DCNv2) to use YOLACT++ version"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "HDrNO2vIxh05",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!python setup.py build develop &> /dev/null"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "rQgYBYmtx-Q-",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 35
+        },
+        "outputId": "5e634356-71e9-4ab8-cea0-daed9f7a13f4"
+      },
+      "source": [
+        "cd /content/yolact/"
+      ],
+      "execution_count": 11,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "/content/yolact\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ATZ5d6B83ptv",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!mkdir uploads\n",
+        "!mkdir results"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p-TWRro498oW",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Upload files from local filesystem"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Js12lLSb35P3",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "from google.colab import files\n",
+        "\n",
+        "uploaded = files.upload()\n",
+        "pic_names = list(uploaded.keys())\n",
+        "for pic_name in pic_names:\n",
+        "  print(pic_name)\n",
+        "  !mv ./$pic_name ./uploads/$pic_name"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dkank5TW-XgL",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Detect objects in the uploaded pictures"
```

Comment:
...as the custom image upload may fail, offer a cell running detection on already available image(s) in the repo.